Lib/test/test_math.py

```diff
@@ -499,17 +499,13 @@ def testFactorial(self):
             self.assertEqual(math.factorial(i), total)
             self.assertEqual(math.factorial(i), py_factorial(i))
         self.assertRaises(ValueError, math.factorial, -1)
-        self.assertRaises(ValueError, math.factorial, -10**100)
+        self.assertRaises((ValueError, OverflowError), math.factorial, -10**100)
```

Comment:
Hmm. Getting an `OverflowError` here seems wrong to me, but perhaps that's something that could be fixed independently of this PR.

Reply:
Resolved. Thank you!